lib/widgets/side_pane.dart

```diff
@@ -88,22 +89,21 @@ class _SidePaneState extends State<SidePane> {
                 ContextMenuItem(
                   child: const Text('Open in new window'),
                   onTap: () async {
-                    await Process.start(
-                      Platform.resolvedExecutable,
-                      [widget.destinations[index].path],
-                    );
+                    await Process.start(Platform.resolvedExecutable, [
+                      widget.destinations[index].path,
+                    ]);
```

Comment:
for future reference: would be nice to migrate to GAppInfo for this: https://docs.gtk.org/gio/iface.AppInfo.html